### DISCUSSION TITLE
Zero-flux outliers

### DISCUSSION THREAD

**Discussion by 0xCoto** [chime-frb-open-data/community#25] [Q&A] [answered]
I have noticed the following events have a reported flux equal to **0 Jy** (and therefore a fluence of **0 Jy ms**):
- [FRB 20190307A](http://chime-frb.ca/catalog/FRB20190307A)
- [FRB 20190307B](http://chime-frb.ca/catalog/FRB20190307B)
- [FRB 20190329B](http://chime-frb.ca/catalog/FRB20190329B)
- [FRB 20190329C](http://chime-frb.ca/catalog/FRB20190329C)
- [FRB 20190531A](http://chime-frb.ca/catalog/FRB20190531A)
- [FRB 20190531B](http://chime-frb.ca/catalog/FRB20190531B)

Possibly an automated flux-derivation issue 'misleadingly' yielding `0` as an error indication?

**Comment by zpleunis** [comment, marked as answer]
Hi @0xCoto, this is intentional. Quoting from section 3.4 of the catalog paper: "A total of 6 bursts were detected directly after a system restart, when we were not able to obtain steady source transits before upstream complex gain calibration was applied. Since we could not measure proper beamformer unit to Jansky scalings during these times, we do not provide fluences or fluxes for these bursts."

**Comment by 0xCoto** [comment]
@zpleunis - Thanks for the input! I've replaced these values with `-` (instead of `0`) in my catalog then.
